eth/db/chain.py

```diff
@@ -29,42 +29,42 @@
     to_tuple,
 )
 
-from eth_hash.auto import keccak
+from eth_hash.evm.auto import keccak
```

Comment:
I think this is an error.  Doesn't appear to be an `eth_hash.evm` module.

Reply:
Yup, thanks!